packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -38,69 +38,85 @@ export const Settings = () => {
+            alt="Your avatar"
+            route="#"
+          />
+
+          <hr className="unit-row-hr" />
+
           <UnitRow
             header="Display name"
             headerValue={account.displayName}
             route="#"
           />
-        )}
-
-        <UnitRow header="Password" headerValue="********" route="#">
-          <p>Created {account.passwordCreated}</p>
-        </UnitRow>
-
-        {/*
-        /* TO DO: primary/secondary email section with
-        /* verified status and number of secondary emails
-        /* taken into account
-        */}
-        <UnitRow header="Primary email" headerValue={primaryEmail.email} />
-
-        <UnitRow
-          header="Secondary email"
-          headerValue={null}
-          {...{
-            revealModal,
-            modalRevealed,
-          }}
-        >
-          <p>Access your account if you can't log in to your primary email.</p>
-          <p>
-            Note: a secondary email won't restore your information—you'll need a{' '}
-            <a href="#recovery-key">recovery key</a> for that.
-          </p>
-
-          {modalRevealed && (
-            <Modal
-              onDismiss={hideModal}
-              onConfirm={onSecondaryEmailConfirm}
-              headerId={modalHeaderId}
-              descId={modalDescId}
-            >
-              <h2 id={modalHeaderId}>Verify primary email first</h2>
-              <p id={modalDescId}>
-                Before you can add a secondary email, you must verify your
-                primary email. To do this, you'll need access to{' '}
-                {primaryEmail.email}
-              </p>
-            </Modal>
-          )}
-        </UnitRow>
+
+          <hr className="unit-row-hr" />
+
+          <UnitRow
+            header="Password"
+            headerValueClassName="tracking-wider"
+            headerValue="••••••••••••••••••"
+            route="#"
+          >
+            <p className="text-grey-400 text-xs mobileLandscape:mt-3">
+              Created {account.passwordCreated}
+            </p>
+          </UnitRow>
+
+          <hr className="unit-row-hr" />
+
+          {/*
+          /* TO DO: primary/secondary email section with
+          /* verified status and number of secondary emails
+          /* taken into account
+          */}
+          <UnitRow header="Primary email" headerValue={primaryEmail.email} />
+
+          <hr className="unit-row-hr" />
+
+          <UnitRow
+            header="Secondary email"
+            headerValue={null}
+            {...{
+              revealModal,
+              modalRevealed,
+            }}
+          >
+            <p className="text-sm mt-3">
+              Access your account if you can't log in to your primary email.
+            </p>
+            <p className="text-grey-400 text-xs mt-2">
+              Note: a secondary email won't restore your information—you'll need
+              a <a href="#recovery-key">recovery key</a> for that.
```

Comment:
Is a11y linter going to yell about this href?

Reply:
No, it only complains about `#`. `#anything` is fine.